src/pages/Pools.tsx

```diff
@@ -159,48 +199,55 @@ function Pools(): ReactElement | null {
       <div className={styles.content}>
         {Object.values(POOLS_MAP)
           .filter(
-            ({ type, migration, isOutdated }) =>
+            ({ type, isOutdated }) =>
```

Comment:
One last thing - since we removed `migration` from the `POOLS_MAP` definition, migrated pools are no longer filtered with the `outdated` filter as they were before. 

Please move this filter statement to somewhere where it can include `isMigrated` in its logic (after where we read `getPropsForPool`) .